python/GafferDispatchTest/SystemCommandTest.py

```diff
@@ -115,6 +115,27 @@ def testFrameRangeSubstitutions( self ) :
 		self.assertEqual( len( sequences ), 1 )
 		self.assertEqual( str( sequences[0] ), "systemCommandTest.####.txt 1-10" )
 
+	def testShell( self ) :
+
+		s = Gaffer.ScriptNode()
+
+		s["n"] = GafferDispatch.SystemCommand()
+
+		self.assertEqual( s["n"]["shell"].getValue(), True )
+
+		# The following command is only valid when interpreted as a shell command
+		s["n"]["command"].setValue( "date | wc -l" )
+
+		d = GafferDispatch.LocalDispatcher()
```

Comment:
I suspect you copied this from elsewhere in this file, but using a dispatcher is overkill when we just want to test a node in isolation like this. We can simply call `s["n"]["task"].execute()`.

Reply:
Addressed in https://github.com/GafferHQ/gaffer/commit/17f5305fb8defc8a78524fe96ca204b26d05b669